.github/workflows/tauri.yaml

```diff
@@ -1,9 +1,8 @@
 name: Tauri builds
 on:
   push:
-    branches:
+    branches-ignore:
       - main
-  pull_request:
```

Comment:
the reason we have this is because otherwise the workflows aren't able to run on PRs from downstream forks